<launch>
    <!-- Navigation Program -->
    <arg name="distance_laser" default="0.3" />
    <arg name="speed_factor" default="0.1" />
    <arg name="forward_speed" default="1" />
    <arg name="lateral_speed" default="1" />

    <node name="wall_follower" pkg="rubot_control" type="rubot_wall_follower_holonomic.py" output="screen">
        <param name="distance_laser" value="$(arg distance_laser)" />
        <param name="speed_factor" value="$(arg speed_factor)" />
        <param name="forward_speed" value="$(arg forward_speed)" />
        <param name="lateral_speed" value="$(arg lateral_speed)" />
    </node>
</launch>